mavros/src/lib/enum_to_string.cpp

```diff
@@ -245,5 +246,55 @@ timesync_mode timesync_mode_from_str(const std::string &mode)
 	return timesync_mode::NONE;
 }
 
+static const std::array<const std::string, 12> frame_strings{{
+	/*  0 */ "GLOBAL",
+	/*  1 */ "LOCAL_NED",
+	/*  2 */ "MISSION",
+	/*  3 */ "GLOBAL_RELATIVE_ALT",
+	/*  4 */ "LOCAL_ENU",
+	/*  5 */ "GLOBAL_INT",
+	/*  6 */ "GLOBAL_RELATIVE_ALT_INT",
+	/*  7 */ "LOCAL_OFFSET_NED",
+	/*  8 */ "BODY_NED",
+	/*  9 */ "BODY_OFFSET_NED",
+	/* 10 */ "GLOBAL_TERRAIN_ALT",
+	/* 11 */ "GLOBAL_TERRAIN_ALT_INT",
+}};
+
+std::string to_string(MAV_FRAME e)
+{
+	size_t idx = enum_value(e);
+	if (idx >= frame_strings.size()) {
+		ROS_ERROR_NAMED("uas", "FRAME: wrong frame index: %zu", idx);
+		return std::to_string(idx);
+	}
+
+	return frame_strings[idx];
+}
```

Comment:
the other arrays and functions were also generated from pymavlink? where can I change that so it can also generate them?